packages/light-client/vite.config.ts

```diff
@@ -1,11 +1,26 @@
+import path from "node:path";
+import fs from "node:fs/promises";
 import {defineConfig, mergeConfig} from "vite";
+import dts from "vite-plugin-dts";
 import {getBaseViteConfig} from "../../vite.base.config.js";
 
 import pkgJSON from "./package.json";
 
 export default mergeConfig(
   getBaseViteConfig(pkgJSON, {libName: "LightClient", entry: "src/index.ts"}),
   defineConfig({
+    plugins: [
+      dts({
+        rollupTypes: true,
+        bundledPackages: ["@lodestar/*", "@chainsafe/persistent-merkle-tree", "@chainsafe/bls", "@chainsafe/ssz"],
```

Comment:
How do you come up with this list? Is there no way to automate this?

Reply:
By building the types buyndle and investigating the types import in it. The ideal scenario we want is there should not be any thirdparty types import in types bundle.